src/themes/components/menu.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  createMultiStyleConfigHelpers,
+  defineStyle,
+} from '@chakra-ui/styled-system';
+import { menuAnatomy as parts } from '@chakra-ui/anatomy';
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(parts.keys);
+
+const baseStyleList = defineStyle({
+  bg: 'white',
+  boxShadow: 'sm',
+  color: 'inherit',
+  minW: '3xs',
+  py: '2',
+  zIndex: 1,
+  borderRadius: 'sm',
+  borderWidth: '1px',
+  borderColor: 'gray.200',
+});
+
+const baseStyleItem = defineStyle({
+  paddingY: 'md',
```

Comment:
in the design there is a hover color for the user menu element that is blue. Was this dropped?
![image](https://user-images.githubusercontent.com/71456764/195004970-d1c3872d-b723-4f73-a545-0f9838d7fb7f.png)

Reply:
It's intended for the language navigation I can have a chat with Sergiu regarding the rotating chevron. For the user navigation, we will make use of the drawer component (and the design will slightly change)
PoC: https://github.com/smg-automotive/components-pkg/pull/187
Designs: https://www.figma.com/file/8vy6tgk0DV9KSKyJBjiLSH/12.-AS24---Web---Patterns?node-id=159%3A5729